sdk/deviceupdate/iot-device-update-rest/test/public/utils/recordedClient.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="esnext.asynciterable" />
+
+import { Context } from "mocha";
+
+import { Recorder, RecorderEnvironmentSetup, env, record } from "@azure-tools/test-recorder";
```

Comment:
It would be great if you could migrate this to the new Recorder? Here is the migration guide https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder-new/MIGRATION.md

New packages are now expected to use the new recorder.

Here's a PR migrating one existing package for your reference: https://github.com/Azure/azure-sdk-for-js/pull/19628/files

Reply:
I guess the new recorder asks me to re-record the test recordings. However, I cannot do it now because the instance for our test is deleted and I prefer to use this previous recorder in this PR.